modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/SelectTag.java

```diff
@@ -0,0 +1,190 @@
+
+	@Override
+	public int doStartTag() throws JspException {
+		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
+
+		if (Validator.isNotNull(getLabel())) {
+			setDynamicAttribute(StringPool.BLANK, "aria-label", getLabel());
+		}
+
+		return super.doStartTag();
+	}
+
+	public String getLabel() {
+		return LanguageUtil.get(
+			TagResourceBundleUtil.getResourceBundle(pageContext), _label);
+	}
+
+	public String getName() {
+		return _name;
+	}
+
+	public List<SelectOption> getOptions() {
+		return _options;
+	}
+
+	public boolean isDisabled() {
+		return _disabled;
+	}
+
+	public boolean isMultiple() {
+		return _multiple;
+	}
+
+	public void setDisabled(boolean disabled) {
+		_disabled = disabled;
+	}
+
+	public void setLabel(String label) {
+		_label = label;
+	}
+
+	public void setMultiple(boolean multiple) {
+		_multiple = multiple;
+	}
+
+	public void setName(String name) {
+		_name = name;
+	}
+
+	public void setOptions(List<SelectOption> options) {
+		_options = options;
+	}
+
+	@Override
+	protected void cleanUp() {
+		super.cleanUp();
+
+		_disabled = false;
+		_label = null;
+		_multiple = false;
+		_name = null;
+		_options = null;
+	}
+
+	@Override
+	protected String getHydratedModuleName() {
+		return "frontend-taglib-clay/Select";
+	}
+
+	@Override
+	protected Map<String, Object> prepareProps(Map<String, Object> props) {
+		props.put("disabled", _disabled);
+
+		if (Validator.isNotNull(getLabel())) {
+			props.put("label", getLabel());
+		}
+
+		props.put("multiple", _multiple);
+		props.put("name", _name);
+		props.put("options", _options);
+
+		return super.prepareProps(props);
+	}
+
+	@Override
+	protected String processCssClasses(Set<String> cssClasses) {
+		cssClasses.add("form-control");
+
+		return super.processCssClasses(cssClasses);
+	}
+
+	@Override
+	protected int processStartTag() throws Exception {
+		super.processStartTag();
+
+		JspWriter jspWriter = pageContext.getOut();
+
+		if (Validator.isNotNull(_label)) {
+			jspWriter.write("<label>");
+			jspWriter.write(_label);
+			jspWriter.write("</label>");
+		}
+
+		jspWriter.write("<select ");
+
+		super.writeDynamicAttributes();
+
+		super.writeCssClassAttribute();
+
+		if (_disabled) {
+			jspWriter.write(" disabled");
+		}
+
+		if (Validator.isNotNull(_name)) {
+			jspWriter.write(" name=\"");
+			jspWriter.write(_name);
+			jspWriter.write("\"");
+		}
+
+		jspWriter.write(" />");
+
+		for (SelectOption option : _options) {
+			jspWriter.write("<option>");
+			jspWriter.write(option.getLabel());
+			jspWriter.write("</option>");
+		}
+
+		jspWriter.write("</select>");
+
+		return SKIP_BODY;
+	}
+
+	@Override
+	protected void writeCssClassAttribute() throws Exception {
+		JspWriter jspWriter = pageContext.getOut();
+
+		jspWriter.write(" class=\"form-group\"");
+	}
+
+	@Override
+	protected void writeDynamicAttributes() {
+	}
+
+	@Override
+	protected void writeIdAttribute() {
+	}
+
+	private static final String _ATTRIBUTE_NAMESPACE = "clay:selecttwo:";
```

Comment:
I guess this is just a test? 😂

Reply:
Yea, I had two select tags just to compare them easier 😄